File: Python/uberVan.py
from car import Car
class UberVan(Car):
    typeCardAcepted = []
    material = []
    def __init__(self, license, driver, typeCardAcepted, material):
        super(UberVan, self).__init__(license, driver)
        self.typeCardAcepted = typeCardAcepted
        self.material = material

File: PHP/UberVan.php
<?php
require_once("Car.php");
class UberVan extends Car {
    public $typeCardAcepted;
    public $material;
    private $passengenger;

    function __construct($license, $driver, $typeCardAcepted, $material) {
        parent::__construct($license, $driver);
        $this->typeCardAcepted = $typeCardAcepted;
        $this->material = $material;
    }

    public function printDataCar() {
        echo "License Car: {$this->license} and model is: {$this->model} pasenger: {$this->passengenger}<br>";
    }

    public function getPasenger() {
        return $this->passengenger;
    }

    public function setPasenger($passenger) {
        if($passenger == 6){
            $this->passengenger = $passenger;
        } else {
            echo "El valor de Passenger: debe ser seis. <br>";
        }
    }

}